src/com/cje/RequestPage.java

```diff
@@ -1,17 +1,633 @@
 package com.cje;
 
+import javax.rmi.CORBA.Util;
 import javax.swing.*;
+import javax.swing.text.MaskFormatter;
+import java.awt.*;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.awt.event.*;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.text.ParseException;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.logging.Level;
 
 /**
- * Created by bbybby on 6/8/2016.
+ * Created by Jieun Choi on 6/8/2016.
+ *
+ * class for User interface
  */
-public class RequestPage {
+public class RequestPage extends JFrame{
     private JTextField titleTextField;
 
-    private JPanel panel1;
+    private JPanel mainPanel;
     private JPanel inputPanel;
     private JRadioButton maleRdBtn;
     private JRadioButton femaleRdBtn;
     private JComboBox birthYearCBox;
-    private JComboBox birthMonthCBox;
+    private JFormattedTextField heightTxt;
+    private JFormattedTextField weightTxt;
+    private JFormattedTextField headTxt;
+    private JFormattedTextField bmiTxt;
+    private JButton viewBtn;
+    private JFormattedTextField birthWeightTxt;
+    private JFormattedTextField birthHeightTxt;
+    private JLabel ageTxt;
+    private JLabel baseDateTxt;
+    private JFormattedTextField birthDateTxt;
+    private JButton resetBtn;
+    private JButton exitBtn;
+    private JPanel resultPanel;
+    private JLabel resultTxt;
+    private JCheckBox calculateByDayCkBox;
+    private JButton changeDateBtn;
+    private JLabel ageLabel;
+
+    private DataManager dm;
+    private ResultView rv;
+    private Params.Gender gender = Params.Gender.NONE;
+    private int birthYear;
+    private int birthMonth;
+    private int birthDay;
+    private LocalDate baseDate;
+
+    private float birthWeight;
+    private float birthHeight;
+    private float height;
+    private float weight;
+    private float head;
+    private float bmi;
+
+    private int ageByDay;
+
+    boolean isDataLoaded;
+    boolean isCalculatedByDay;
+
+    // Initialize member variables
+    private void initParam() {
+        birthYear = -1;
```

Comment:
do you really need this initialization? I mean do you use `-1` as a special number somewhere?
If not, please remove this. This makes reader confused.
When I looked at this, I thought you may have special logic when `birthYear == -1`, but I couldn't find any.

Also, if this is really required, you can initialize this when you define this variable.